next/components/ui/Timeline/Timeline.tsx

```diff
@@ -0,0 +1,47 @@
+import Markdown from '@components/atoms/Markdown'
+import cx from 'classnames'
+import { VerticalTimeline, VerticalTimelineElement } from 'react-vertical-timeline-component'
+
+type TimelineItemProps = {
+  id: string
+  title?: string | null
+  content?: string | null
+}
+
+type TimelineProps = {
+  timelineItems: TimelineItemProps[]
+}
+
+export const Timeline = ({ timelineItems }: TimelineProps) => {
+  return (
+    <VerticalTimeline animate={false} lineColor="rgb(var(--color-category-600))">
+      {timelineItems.map((timelineItem, index) => (
+        <VerticalTimelineElement
+          key={timelineItem.id}
+          className={cx('vertical-timeline-element--borderless', {
+            'vertical-timeline-element--with-image': index === 1 || index === 2,
```

Comment:
Please remove `vertical-timeline-element--borderless` and `vertical-timeline-element--with-image` and replace with tailwind classes.